gguf-py/gguf/constants.py

```diff
@@ -1057,7 +1068,17 @@ class MODEL_TENSOR(IntEnum):
     MODEL_TENSOR.V_MM_POST_NORM:            "mm.post_norm",
     MODEL_TENSOR.V_MM_INP_PROJ:             "mm.input_projection",
     MODEL_TENSOR.V_MM_INP_NORM:             "mm.input_norm",
-    MODEL_TENSOR.V_MM_SOFT_EMB_NORM:        "mm.soft_emb_norm",
+    MODEL_TENSOR.V_MM_SOFT_EMB_NORM:        "mm.soft_emb_norm",             # gemma3n
+    MODEL_TENSOR.V_MM_EMBEDDING:            "mm.embedding",                 # gemma3n
+    MODEL_TENSOR.V_MM_HARD_EMB_NORM:        "mm.hard_emb_norm",             # gemma3n
+    MODEL_TENSOR.V_ENC_CONV_STEM:           "v.conv_stem.conv",         # gemma3n
+    MODEL_TENSOR.V_ENC_CONV_STEM_BIAS:      "v.conv_stem.conv.bias",    # gemma3n
```

Comment:
the conversion script can automatically detect postfix `.weight` and `.bias`, there is no need to define this manually


```suggestion
    MODEL_TENSOR.V_ENC_CONV_STEM:           "v.conv_stem.conv",         # gemma3n
```

If you need to handle `_bias` case, do a quick Ctrl+F and look around to see how other models handle it